### ISSUE TITLE
Redesign "Projects Joined"-page

### ISSUE THREAD

**Issue by gedankenstuecke** [OpenHumans/open-humans#740]
## General Checkups
- [x] Have you checked that there isn't already an existing issue that describes what you report below?
- [x] Have you checked that there isn't already an open pull requests for this issue/update/change?

## Description
Right now `https://www.openhumans.org/member/me/joined/` and `https://www.openhumans.org/member/me/data/` are confusing and lacking for a number of reasons: 

1. Projects which neither ask for data access nor data uploads are listed on neither page and just are basically invisible
2. Projects which are under development are absent too, making it hard to leave these
3. Projects which want to upload data but haven't done so are listed under the `data` page. 

Generally, it's hard to predict where each project will land and it's hard for users to find out which projects they've joined and which data access they've consented to. For this reason this page should be redesigned. The ideas so far

Make `/member/me/joined/` page that 
- lists all projects that a member has joined
    - including these that are under development 
    - request neither access nor upload to/of data
    - those who want to upload data

Generally the idea is that everything that a member has opted in into should be listed. At the same time this page should list the permissions that people have agreed to. 

## Example
@madprime made a suggestion in a google doc for how this could look like: https://docs.google.com/document/d/18_PsFnKKMMDZuJ_BqyVjVRKz14jWns3pSiSG6LD3IOA/edit

My idea was to use the badges of each project instead to visualize the permissions granted and to gray out those data sources that are not applicable to a given member right now (e.g. project X asks for data source Y but you haven't connected Y yet). A quick drawing is below.

![image](https://user-images.githubusercontent.com/674899/37435324-0e6033ca-27a0-11e8-8b51-43f48aefffdf.png)

**Comment by gedankenstuecke** [comment]
Related to this (and important for GDPR related overhauls, ping @jhdulaney) The `/member/me/data` slug shows only data that was deposited by projects that have been set to `approved`, data from unapproved data sources is not listed. Along with the problems listed above this makes it ± impossible to delete and find these data.

**Comment by jhdulaney** [comment]
Yeah, I noticed that.  That may be a first thing to look at here.
